### PR TITLE
updates to works

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: 5001-6000/LIT5518DiscourseofTheodore.xml [hunk 1]
```diff
@@ -49,8 +49,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
         <body>
-            <div type="bibliography"/>
-         <!---->
+            <div type="edition" xml:lang="gez">
+                <div type="textpart" subtype="incipit"><ab>ድርሳን፡ ዘነበበ፡ አቡነ፡ ቅዱስ፡ ወክቡር፡ በዘዚአሁ፡ ጾታ፡ አባ፡ ቴዎድሮስ፡ ሊቀ፡ ጳጳሳት፡ ዘሀገረ፡ አንጾኪያ። 
+                    በእንተ፡ ክብሩ፡  ለቅዱስ፡ ጽኑእ፡ ወኃያል፡ ድንግል፡  መዋኢ፡ መስተጋድል፡ መልክአ፡ መላእክተ፡  እግዚእ፡ 
+                    ገላውዴዎስ፡ ውልደ፡ ንጉሥ፡ መፍቀሬ፡ እግዚአብሔር፡ አብጥልማዎስ። 
+                    ወአኃዘ፡ ተጋድሎ፡  እመአመ፡ ንዕሱ፡ እስከ፡ ተፍጻሜቱ፡  ገድሉ፡  ክብርት፡  ዘውእቱ፡  አመ፡ ፲ወአሚሩ፡ ለወርኃ፡ ሰኔ። ወከሠተ፡ 
+                    ዓዲ፡ ክብረ፡ ዘመዱ፡ ወተሰዶቱ፡ ኀበ፡ ስዒድ፡ ምስር።</ab></div>
+                <note>After BL Orient 700</note>
```
Comment (thea-m) [inline 5001-6000/LIT5518DiscourseofTheodore.xml]: could you mark up the manuscript?

File: 5001-6000/LIT5547DiscourseConstantine.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">Discourse of Constantine</title>
+                <title xml:lang="gez" xml:id="t1">Discourse of Constantine on the miracles of Claudius</title>
```
Comment (thea-m) [inline 5001-6000/LIT5547DiscourseConstantine.xml]: en

File: 5001-6000/LIT5518DiscourseofTheodore.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">Discourseof Theodore, patriarch of Antioch</title>
+                <title xml:lang="gez" xml:id="t1">Discourse of Theodore, patriarch of Antioch, on Claudius, the son of Ptolemy</title>
```
Comment (thea-m) [inline 5001-6000/LIT5518DiscourseofTheodore.xml]: en

File: 5001-6000/LIT5547DiscourseConstantine.xml [hunk 4]
```diff
@@ -45,12 +45,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                    ለባሴ፡ ክርስቶስ፡ ወዑጹፍ፡ በትሩፋተ፡ መንፈስ፡ ቅዱስ፡ ዘምጉብ፡
+                    በኃይለ፡ አምላካዊት፡ አባ፡ ቈስጠንጢኖስ፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ አስዩጥ፡ ዘተናገረ፡ ዘንተ፡
+                    በእንተ፡ ኃይል፡ ወጽንዕ፡  መስተጋደል፡ ወመዊዕ፡ ድንግል፡ ወንጹሕ፡ ፍቁረ፡ 
+                    ንጉሥ፡ ዐቢይ፡ ሰማዕት፡ ክቡር፡ ገላውዴዎስ፡ ወልደ፡
+                    ንጉሥ፡ መፍቀሬ፡ እግዚአብሔር፡ አብጥልማዎስ። ወሠርዖ፡ ለዝንቱ፡ ውስተ፡ ቤተ፡ ክርስቲያኑ፡ 
+                    ንጽሕት፡   እንተ፡  ሀለወት፡ በሀገረ፡ አስዩጥ። ወኮኑ፡ ጉቡኣነ፡ በዘ፡ በዓል፡ ሰብአ፡ ፫አህጉር።  እለ፡ እሙንቱ፡ 
+                    ሰብአ፡ ሀገር፡ እንጽና።  ወሰብአ፡ ሀገረ፡ አስሙናይን። ወሰብአ፡ ሀገረ፡ አስዩጥ፡ </ab></div>
+                <div type="textpart" subtype="colophon"><ab>ወለዝንቱ፡ ገድል፡ ተርጐሞ፡ እምልሳነ፡ ቅብጢ፡ ኀበ፡ ዓረቢ፡ አብ፡ ክቡር፡ ወመምህር፡
+                    ትሩፍ፡ ጥቢብ፡ ወፍጹም፡ ኤጲስ፡ ቆጶስ፡ ዘሀገር፡ አባይ፡ እስዩጥ፡ ወኲሉ፡ አድያሚሃ፡ ዘዓውደ። </ab></div>
+                <note>After BL Orient 700</note>
```
Comment (thea-m) [inline 5001-6000/LIT5547DiscourseConstantine.xml]: see above

File: 5001-6000/LIT5518DiscourseofTheodore.xml (hunk 1 above)
Comment (eu-genia) [reply 5001-6000/LIT5518DiscourseofTheodore.xml]: sure

File: 5001-6000/LIT5547DiscourseConstantine.xml (hunk 4 above)
Comment (eu-genia) [reply 5001-6000/LIT5547DiscourseConstantine.xml]: all done

**Comment by thea-m** [comment]
I meant marking up the reference to the manuscript, not the manuscript itself, in case there was confusion. `<ref type="mss" corresp="BLorient700"> `. Sorry for being unclear.
But are you also working on Wright's catalogue now? Then it would be good if we coordinate our work so that we don't accidentally work in the same manuscript in individual branches the other isn't aware of.

**Comment by eu-genia** [comment]
Sorry, done.

I now did BL orient 774  (which had a stub but no content) because it was nearly identical to ESap005, and otherwise I only touch upon them when coming across texts or persons I need to fix for other purposes like now with BLor700 - but yes I am ready to switch to BL; it is not clear to me what to do with the stubs created by Solomon as they are not really useful to my mind - any suggestions?)

**Comment by thea-m** [comment]
I am working through the catalogue from the beginning (well, after the records you did in the beginning), so when I come to one which is such a stub (there were a few already) I simply continue the description. It does save some time not having to create a record.

**Pull request merged**